src/KubernetesClientConfiguration.ConfigFile.cs

```diff
@@ -57,7 +57,7 @@ public partial class KubernetesClientConfiguration
 
         /// <summary>
         /// </summary>
-        /// <param name="kubeconfig">Fileinfo of the kubeconfig, cannot be null, whitespaced or empty</param>
+        /// <param name="kubeconfig">String representation of kubeconfig contents, cannot be null, whitespaced or empty</param>
         /// <param name="currentContext">override the context in config file, set null if do not want to override</param>
         /// <param name="masterUrl">overrider kube api server endpoint, set null if do not want to override</param>
         public static KubernetesClientConfiguration BuildConfigFromConfigFile(string kubeconfig,

```

Comment:
I couldn't find any references to this method. Can we deprecate?

Reply:
Sure, seems reasonable.

Reply:
Mark as [Obsolete] or remove?